nucypher/policy/policies.py

```diff
@@ -402,13 +392,13 @@ def _consider_arrangements(self,
                                                         arrangement=arrangement,
                                                         network_middleware=network_middleware)
 
-            except NodeSeemsToBeDown:  # TODO: #355 Also catch InvalidNode here?
+            except NodeSeemsToBeDown as e:  # TODO: #355 Also catch InvalidNode here?
```

Comment:
That's `e` for a debugger, right?

Reply:
Ahh yes.  It can go if you want, but it's an awfully useful place to have one if you're using a debugger.